pkg/plugins/keptn.go

```diff
@@ -0,0 +1,167 @@
+package plugins
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"code.gitea.io/sdk/gitea"
+	"github.com/davecgh/go-spew/spew"
+)
+
+const (
+	KeptnPluginName = "keptn"
+
+	GitURLParam      = "url"
+	GitUsernameParam = "username"
+	GitPasswordParam = "password"
+	GitAPITokenParam = "git-token"
+
+	GitDefaultURL      = "http://orkestra-gitea-http.orkestra.svc.cluster.local:3000"
+	GitAdminUsername   = "gitea_admin"
+	GitDefaultUsername = "keptn"
+	GitDefaultPassword = "password"
+)
+
+type Git struct {
+	URL      string
+	Username string
+	Password string
+
+	APIToken string
+}
+
+// Keptn implements the plugin interface
+type Keptn struct {
+	git Git
+}
+
+func keptn() *Keptn {
```

Comment:
Better name, probably just `NewKeptn()` or something, general practice is that when we create the object, we do so with `New<thing>`